app/controllers/graphql_controller.rb

```diff
@@ -9,13 +9,14 @@
 # @see http://graphql-ruby.org/schema/generators#graphqlinstall
 #
 class GraphqlController < ApplicationController
+  skip_before_action :verify_authenticity_token
```

Comment:
have lots of requests coming from front_end and see no real need for csrf token checking for this